src/TestExplorer/TestRunner.ts

```diff
@@ -233,9 +233,10 @@ export class TestRunner {
                 token,
                 {
                     cwd: testBuildConfig.cwd,
-                    env: { ...process.env, ...testBuildConfig.env },
+                    env: { ...testBuildConfig.env, ...process.env },
```

Comment:
Why have you swapped these two around. This now means variables in `process.env` override the variables from `testBuildConfig.env`